stock_landed_costs_purchase_auto/tests/common.py

```diff
@@ -31,7 +31,9 @@ def setUpClass(cls):
             }
         )
         cls.purchase_user = new_test_user(
-            cls.env, login="test_purchase_user", groups="purchase.group_purchase_user"
+            cls.env,
+            login="test_purchase_user",
+            groups="purchase.group_purchase_user,stock.group_stock_user",
```

Comment:
It shouldn't be necessary to add stock.group_stock_user to test the process, right?